### PR TITLE
feat: Add transformer role and some chore actions

### PR DESCRIPTION
**Pull request by jakubigla**

### CODE REVIEW

**Review by dgniewek — APPROVED** [review]
Looks good

File: main.tf [hunk 1]
```diff
@@ -27,49 +27,55 @@ module "snowflake_default_role" {
...
-
 module "snowflake_custom_role" {
   for_each = local.custom_roles
 
   source  = "getindata/role/snowflake"
   version = "1.0.3"
-  context = module.this.context
-  enabled = module.this.enabled && lookup(each.value, "enabled", true)
+
+  context         = local.enabled
```
Comment (Santhin) [inline main.tf:50]: Shouldn't we pass here `module.this.context`?

File: locals.tf [hunk 2]
```diff
@@ -5,28 +5,53 @@ locals {
     lookup(module.database_label.descriptors, var.descriptor_name, module.database_label.id), "/${module.database_label.delimiter}${module.database_label.delimiter}+/", module.database_label.delimiter
   ), module.database_label.delimiter) : null
 
+  enabled              = module.this.enabled
```
Comment (Santhin) [inline locals.tf:8]: What is the purpose of this var if it's not being used anywhere?

File: main.tf [hunk 3]
```diff
@@ -27,49 +27,55 @@ module "snowflake_default_role" {
...
 module "snowflake_schema" {
-  for_each = var.schemas
+  for_each = local.schemas
 
   source  = "getindata/schema/snowflake"
-  version = "1.1.0"
+  version = "1.3.0"
+
   context = module.this.context
+  enabled = each.value.enabled
```
Comment (Santhin) [inline main.tf:70]: Would be good to combine it with `module.this.enabled` or with `local.enabled`?

**Pull request merged**